src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        Colors color = ColorSupplier.getRandomColor();
+        Random random = new Random();
+        int number = random.nextInt(100) + 1;

```

Comment:
magic numbers

Reply:
not fixed
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/c6509f75-8221-4a9e-ab4c-d47d2a7346cc)